honeybadgermpc/ntl/ntlwrapper.h

```diff
@@ -6,4 +6,13 @@
 #include <NTL/mat_ZZ_p.h>
 #include <NTL/BasicThreadPool.h>
 using namespace NTL;
+
+unsigned char* bytesFromZZ(const ZZ& a) {
+   long n = NumBytes(a);
+   // An extra byte read
```

Comment:
Add a comment so as to why this is needed?

Reply:
Sure.